json-rpc/src/views.rs

```diff
@@ -69,36 +69,42 @@ pub enum EventDataView {
         metadata: BytesView,
     },
     SystemEvent,
+    UnknownEvent {},
```

Comment:
nit: why need bracket syntax here? ` {}`
also how SystemEvent and UnknownEvent are really different. I feel like we should just keep one. Which one do you want to pick?

Reply:
UnknownEvent seems more general, let me delete SystemEvent

Reply:
Also, we should use rename to detonate exact name here for clarity